### PR TITLE
[AFD] WARNING: S-GBS OUTBREAK DETECTED

### PR DESCRIPTION
**Pull request by Contrabang**
## What Does This PR Do

An S-GBS Outbreak has been detected at this Nanotrasen Station!

(Gives everyone a modfified version of the gib-self verb).

Captain also gets an alert of the S-GBS outbreak

## Why It's Good For The Game
yes

## Images of changes

image

## Testing

yes

## Changelog
:cl:
experiment: An S-GBS outbreak has been detected on station. Ask your captain for more details!
/:cl:

### CODE REVIEW

File: code/modules/mob/living/living.dm [hunk 1]
```diff
@@ -1186,3 +1186,30 @@
...
+		var/confirm_antag = tgui_alert(src, "Are you absolutely sure? If you do this after you got converted/joined as an antagonist, you could face a jobban!", "Confirm Suicide", list("Yes", "No"))
+		if(confirm_antag != "Yes")
+			return
+		is_antagonist = TRUE
+	create_log(ATTACK_LOG, "Gibbed themselves (S-GBS)[is_antagonist ? " as a special role" : ""].")
+	if(is_antagonist)
+		log_admin("[key_name(usr)] gibbed themselves (S-GBS) as a special role.")
+		message_admins("[key_name_admin(usr)] gibbed themselves (S-GBS) as a special role.")
+
+	src.gib()
```
Comment (GDNgit) [inline code/modules/mob/living/living.dm:1214]:
```suggestion
	src.gib()
```
lock your doors

Comment (Contrabang) [reply code/modules/mob/living/living.dm:1214]: a small price to pay

File: code/modules/mob/living/living.dm [hunk 2]
```diff
@@ -1186,3 +1186,30 @@
...
+/mob/living/verb/sgbs_infected() // did you know, everyone is infected with S-GBS. Wonder if anyone will notice this in game.
+	set name = "Gib-self"
+	set desc = "Huh, I feel funny. A little explosive maybe."
+	set category = "Special Verbs"
+
+	if(stat != CONSCIOUS)
+		to_chat(src, "<span class='warning'>You can't do this while you're not conscious.</span>")
+		return
+
+	var/confirm = alert(src, "You sure?", "Confirm", "Yes", "No")
```
Comment (AyIong) [inline code/modules/mob/living/living.dm:1199]:
```suggestion
	var/confirm = tgui_alert(src, "You sure?", "Confirm", list("Yes", "No"))
```

Comment (Contrabang) [reply code/modules/mob/living/living.dm:1199]: but the soul

Comment (AyIong) [reply code/modules/mob/living/living.dm:1199]: > but the soul

You're right

**Review by Burzah — APPROVED** [review]

**Pull request merged**